### PR TITLE
Fixes the JWT article to be technically correct again

### PR DESCRIPTION
**Pull request by voidmain**
I changed many of the misnomers in the article and also provided some SEO keywords in a paragraph describing these incorrect names.

### CODE REVIEW

File: site/learn/expert-advice/tokens/jwt-components-explained.md [hunk 1]
```diff
@@ -10,10 +10,11 @@ dateModified: 2022-02-24
 ---
 
 
-A JSON Web Token, or JWT, is an authentication token typically used with standard authentication protocols such as Oauth2. This article explains the component parts of a JWT token, and helps explain how the JWT works. 
+A JSON Web Token (commonly shortened to JWT), is a token typically used with standard protocols such as OAuth2. This article explains the component parts of a JWT, and helps explain how the JWT works.
```
Comment (robotdan) [inline site/learn/expert-advice/tokens/jwt-components-explained.md:13]: Should we mention a pronunciation of "jot" ?

**Review by robotdan — COMMENTED** [review]
LGTM

File: site/learn/expert-advice/tokens/jwt-components-explained.md [hunk 2]
```diff
@@ -1,7 +1,7 @@
 ---
 layout: advice
-title: Components of JWT Authentication Tokens Explained
-description: Technical authentication article explains JSON Web Tokens (JWT) and their component parts. 
+title: Components of JWTs Explained
```
Comment (jobannon) [inline site/learn/expert-advice/tokens/jwt-components-explained.md:3]: Also need update the title on the brochure site.

learn/expert-advice/tokens/index.html:14

```
{% include _advice-card.liquid href="/learn/expert-advice/tokens/jwt-authentication-token-components-explained" img="/assets/img/stamps/anatomy-of-a-jwt-stamp.svg" title="Components of JWT Authentication Tokens Explained" description="JSON Web Tokens (JWTs) look like gibberish. But they are actually quite structured, with a header, payload and signature." %}
```

**Review by jobannon — COMMENTED** [review]
One link change

File: site/learn/expert-advice/tokens/jwt-components-explained.md [hunk 3]
```diff
@@ -173,9 +174,9 @@ While you can read the [relevant portion of the specification](https://datatrack
 
 When the JWT is received, the same operations can be performed. If the generated signature is correct, the contents of the JWT are unchanged from when it was created.
 
-## JWT Authentication Token Limits
+## Limits
```
Comment (mooreds) [inline site/learn/expert-advice/tokens/jwt-components-explained.md:177]:
```suggestion
## JSON Web Token Limits
```

**Review by mooreds — APPROVED** [review]

**Pull request merged**